Library/Homebrew/cask/cask_loader.rb

```diff
@@ -50,10 +50,10 @@ def cask(header_token, **options, &block)
 
     # Loads a cask from a string.
     class FromContentLoader < AbstractContentLoader
-      def self.can_load?(ref)
+      def self.try_new(ref, warn: false)
         return false unless ref.respond_to?(:to_str)
 
-        content = ref.to_str
+        content = T.unsafe(ref).to_str
```

Comment:
Yes, unfortunate, but it does make sense: https://sorbet.org/docs/flow-sensitive#what-about-respond_to

You cannot deduce a type by its methods alone. We could just use `is_a?(String)` here though, because anything else doesn't make sense anyways.

Reply:
Either way is fine for me. Thanks for explaining it to me.